src/js/components/help/more-help-resources.js

```diff
@@ -1,62 +1,66 @@
 import React from 'react';
 
-export default class MoreHelp extends React.Component {
-  render() {
-    var support = this.props.isHosted ? 'mailto:support@hosted.mender.io' : 'mailto:contact@mender.io';
-
-    return (
-      <div>
-        <h2>More resources</h2>
-
-        <p>{`If you can't find what you're looking for here, there are other places to find help with Mender:`}</p>
-
-        <ul>
-          <li>
-            <p>Ask questions and find answers at the{' '}
-              <a href="https://hub.mender.io" target="_blank">
-                Mender Hub community forum
-              </a>
-            </p>
-          </li>
-
-          <li>
-            <p>Report a bug using{' '}
-              <a href="https://tracker.mender.io" target="_blank">
-                our Bug Tracker
-              </a>
-            </p>
-          </li>
-
-          <li>
-            <p>Read the{' '}
-              <a href={`https://docs.mender.io/${this.props.docsVersion}`} target="_blank">
-                Mender documentation
-              </a>
-            </p>
-          </li>
-        </ul>
-        
-        <br/>
-        <p>Other ways to get in touch:</p>
-        <ul>
-          <li>
-            <p>
-              <a href={support} target="_blank">
-                Email us
-              </a>{' '}
-              with a question
-            </p>
-          </li>
-          <li>
+const MoreHelp = ({ docsVersion, isHosted }) => {
+  var support = isHosted ? 'mailto:support@mender.io' : 'mailto:contact@mender.io';
+  const resources = [
+    {
+      pretext: 'Ask questions and find answers at the',
+      target: 'https://hub.mender.io',
+      linkTitle: 'Mender Hub community forum'
+    },
+    {
+      pretext: 'Report a bug using',
+      target: 'https://tracker.mender.io',
+      linkTitle: 'our Bug Tracker'
+    },
+    {
+      pretext: 'Read the',
+      target: `https://docs.mender.io/${docsVersion}`,
+      linkTitle: 'Mender documentation'
+    },
+    {
+      pretext: 'Ask questions and find answers at the',
```

Comment:
Mender Hub section appears twice, both 1st and last in the list

Reply:
🙈 sorry for that - should now be fixed